config/sidekiq.yml

```diff
@@ -63,7 +63,7 @@
     class: Scheduler::SuspendedUserCleanupScheduler
   status_stat_update_scheduler:
     queue: scheduler
-    interval: '1h'
+    interval: '30m'
```

Comment:
Did you mean to leave this change in here, or was it temporary for staging/testing?

Reply:
yeah. I feel we should be able to hold that at every 30 minutes. Staging didn't seem to have a problem with it, so prod should chew through it just as quickly